GETTING_STARTED.md

```diff
@@ -0,0 +1,128 @@
+# Getting Started
+
+Euphony is very easy to use
+
+This consists of two parts.
+
+1) Transmitter
+ - Generate sound waves and transmits through a microphone.
+2) Receiver
+ - Listen to sound waves and receive data.
```

Comment:
```suggestion
Euphony is a library that transmits data using critical bandwidths of the tonal frequency inaudible to humans.  
Since the API is written in Java, users can use it in any language running on JVM like Java or Kotlin.

This consists of two parts.

1) Transmitter
 - Generate sound waves and transmits through a speaker.
2) Receiver
 - Listen to sound waves and receive data through a MIC.
```